Dockerfile-prod

```diff
@@ -0,0 +1,27 @@
+FROM quay.io/sameersbn/ubuntu:latest
+MAINTAINER Alexander Trost <galexrt@googlemail.com>
+
+ENV ZULIP_VERSION="1.3.10" DATA_DIR="/data"
```

Comment:
For `ZULIP_VERSION`, would it make sense to use `zulip-server-latest.tar.gz` by default but have a way for the user to pass in a different version as an override?  Then we don't need to update this when we do Zulip releases.